### DISCUSSION TITLE
Lost master local connection 'Failed to connect'

### DISCUSSION THREAD

**Discussion by Disincentivato** [marq24/ha-senec-v3#103] [Q&A]
Hi,
from yesterday my master unit is not reachable from the local network.
I can see all the data from the app and no error on the display, I also can ping the unit.
I tried delete the unit and it doesn't connect anymore, I get 'failed to connect ' error.
I reinstalled the integration, restarted the modem and restarted HA. I haven't solved it yet.
The secondary unit is connected.
Is there anything I can check to fix this?
thanks a lot for the support.

![image](https://github.com/user-attachments/assets/ce01b2dc-1741-41df-8eea-d8330aa979b3)
![image](https://github.com/user-attachments/assets/98751995-3901-4fcc-a5eb-714b8eee3436)
![image](https://github.com/user-attachments/assets/6679f549-ef7a-49c6-8225-68ef90aca0df)

**Comment by marq24** [comment]
I am sorry - for what ever reason I don't received notifications about new Questions...

What when you open with a webbrowser
https://192.68.1.20
